integration-tests/api/__tests__/categories/category.ts

```diff
@@ -0,0 +1,85 @@
+import path from "path"
```

Comment:
Since there is a lot of magic going on in the model with typeorm's tree decorators, I'm adding this spec here. 

It feels weird adding it in the api folder, we probably need to move this and similar specs to a more relevant folder. cc. @adrien2p

Reply:
Yeah it is but it is preferable to have it tested than not just because of the folder name :D I did the same in the transaction branch @olivermrbl and in we will have to maybe move it at some point but at least it is tested.